test/type.query.test.js

```diff
@@ -124,6 +124,18 @@ describe('type: query', () => {
             return;
         });
         it('Should change the key during update with --changeKeyValue');
+        it('Should deploy and execute with --execute', async () => {
+            handler.setOptions({ execute: true });
+            // WHEN
+            await handler.deploy('testInstance/testBU', ['query'], ['testNew_query']);
+            // THEN
+            assert.equal(
+                process.exitCode,
+                false,
+                'deploy with --execute should not have thrown an error'
+            );
+            return;
+        });
```

Comment:
I would suggest adding the test for amount of api calls to verify that this did in fact run execute